Makefile

```diff
@@ -26,7 +26,7 @@ GRPC_GATEWAY_VERSION ?= 2.25.1
 VTPROTOBUF_VERSION ?= 0.6.0
 GOIMPORTS_VERSION ?= 0.29.0
 DEEPCOPY_VERSION ?= v0.5.6
-GOLANGCILINT_VERSION ?= v1.64.4
```

Comment:
is it some manual change bypassing kres?

Reply:
Yes, golangci lint had a weird bug on some code, but looks like it no longer reproduces on the last version of code. Anyway I created a bump for kres https://github.com/siderolabs/kres/pull/496